src/frontend/api.spec.js

```diff
@@ -4,7 +4,7 @@ import nodejs from "nodejs-mobile-react-native";
 import RNFS from "react-native-fs";
```

Comment:
@gmaclennan I'm not sure what happened, but it seems like these tests are now failing and timing out? I just uncommented them out.